docs/installation.rst

```diff
@@ -69,17 +54,31 @@ Now you can run installer against your desired host following instructions
 in the `ansible-pulp <https://github.com/pulp/ansible-pulp>`__ installer.
 
 
-Install ``createrepo_c`` from source
-************************************
+Pip install
+-----------
+
+
+Install ``pulpcore``
+********************
+
+Follow the `installation
+instructions <https://docs.pulpproject.org/en/3.0/nightly/installation/instructions.html>`__
+provided with pulpcore.
+
+Install prerequisites
+*********************
+
+Install build dependencies
+##########################
 
-``pulp_rpm`` depends on a Python package named ``createrepo_c``, which is compiled from a C
+``pulp_rpm`` depends on Python packages named ``createrepo_c``, which is compiled from a C
```

Comment:
Probably change this to 

> pulp_rpm depends on some C libraries that must be compiled from source as Python extensions.

And propogate the plural "packages" throughout the paragraph.